include/elliptics/packet.h

```diff
@@ -1000,6 +1000,10 @@ static inline void dnet_convert_iterator_request(struct dnet_iterator_request *r
 	dnet_convert_time(&r->time_end);
 }
 
+
+/* Common record flags */
+#define DNET_RECORD_FLAGS_UNCOMMITTED	(1<<0) /* The record was prepared but hasn't been committed */
```

Comment:
Could we map this one-to-one to blob flags?
I think we might want to get other flags (like ext header or structured format) too someday